packages/react/src/Alert/Alert.tsx

```diff
@@ -20,10 +20,10 @@ export type AlertProps = {
   /** The text for the Heading. */
   heading?: string
   /**
-   * The hierarchical level of the Heading within the document.
-   * Note: this intentionally does not change the font size.
```

Comment:
Don't we want to keep this line? I can imagine this is confusing for users.

Reply:
Rephrased to “Note: the font size is intentionally the same for every level.” and included it for Alert, Accordion, Form Error List and Table of Contents.

Reply:
Ha, this is probably nitpicky, but... I like the old phrasing better. Saying this prop does not change the font size is clearer imo than saying the font size is the same for every level. Because it's not, different levels can have different font sizes, if the `size` prop is also set. `level` just doesn't change anything there.

Reply:
No worries, reverted to the initial wording.